component-interaction/src/app/user-details/user-details.component.ts

```diff
@@ -10,4 +10,11 @@ import { UserModel } from '../user.model';
 export class UserDetailsComponent {
 	@Input()
 	public user?: UserModel;
+
+	@Output()
+	public deleteClick = new EventEmitter<number>();
```

Comment:
```suggestion
	public onDeleteClick = new EventEmitter<number>();
```
we have a convention:
eventEmitters have "on" prefix
handler functions have "handle" prefix